### PR TITLE
Add DISCLAIMER.md

### PR DESCRIPTION
**Pull request by Spaxly**
Good for clarifications

### CODE REVIEW

**Review by justchokingaround — CHANGES_REQUESTED** [review]
change gogoplay to himovies

File: DISCLAIMER.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+<h2>DMCA and Copyright Infrigements</h3>
+
+<br>
+
+<b>
+A browser is a tool, and the maliciousness of the tool is directly based on the user.
+</b>
+
+This project uses client-side content access mechanisms. Hence, the copyright infrigements or DMCA in this project's regards are to be forwarded to the associated site by the associated notifier of any such claims. As of writing this is gogoplay
```
Comment (justchokingaround) [inline DISCLAIMER.md:35]: here

Comment (Spaxly) [reply DISCLAIMER.md:35]: lol forgot about that part

**Review by Spaxly — COMMENTED** [review]
.

**Pull request merged**